ChangeLog.md

```diff
@@ -17,6 +17,13 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- Remove hacks from `memset` handling, in particular, remove it from
+  `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE` and ifdef it out in the wasm backend case
```

Comment:
the phrase "ifdef it out..." to the end of the sentence is a bit unclear. Maybe focus on the wasm backend case, something like "remove it from the JS library (it can be linked from compiler_rt if needed)".

Reply:
Thanks, fixing.